pyproject.toml

```diff
@@ -33,7 +33,10 @@ dependencies = [
     "pydantic-settings",
     "safir[db,uws]>=6.4.0",
     "structlog",
+    "vo-models>=0.4.1",
 ]
+
+
```

Comment:
This doesn't look right: vo-cutouts doesn't use vo-models directly so shouldn't have a dependency on it. Was there a problem you saw when this dependency was missing? It should be a dependency of `safir[uws]`.

Reply:
My bad that slipped in while I was trying to figure out the typing error caused by `pydantic_xml.mypy`